src/components/custom/other-platforms.js

```diff
@@ -237,6 +237,7 @@ export const NavPlatform = ({ onClick, is_ppc, is_ppc_redirect }) => {
             <Flex direction="column" wrap="wrap" jc="flex-start">
                 <StyledText>{localize('Trading platforms')}</StyledText>
                 <NavCard
+                    as="span"
```

Comment:
I don't think you can use the `as` prop here. You can either add `.attrs({ as: 'span' })` directly onto the global `NavCard` and `CardLink` component, or you could extend those onto this page here and add `.attrs({ as: 'a' })` to it.

Reply:
Actually the `as` prop is passed to NavCard -> ResponsiveHeader -> Header . Technically it works but I think it's better to change the prop name to `title_as` cause it's kinda funny to suddenly see an `as` prop in a Card component? 😂